LogsHandler/src/main/java/boot/EntryPoint.java

```diff
@@ -0,0 +1,119 @@
+package boot;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.UriInfo;
+
+import com.google.gson.JsonObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.action.search.SearchRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import Producer.KafkaPublisher;
+import util.InfraUtil;
+
+@Path("entry-point")
+public class EntryPoint {
+    public static Logger logger = LogManager.getLogger(EntryPoint.class);
+    private static int count = 0;
+    private final ElasticSearchHandler elasticsearchClient;
+    private final KafkaPublisher kafkaPublisher;
+
+    @Inject
+    public EntryPoint(ElasticSearchHandler httpHandler ,KafkaPublisher kafkaPublisher) {
+        System.out.println("new Entry point has created");
+        this.elasticsearchClient = httpHandler;
+        this.kafkaPublisher = kafkaPublisher;
+    }
+
+    //for internal server test
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("test")
+    public String test() {
+        System.out.println("logshandler .EntryPoint.test");
+        return "logshandler . my server is runing fine";
+    }
+
+    // indexing constant massage to "index"
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("index")
+    public String sendToProducerConstant() {
+
+        System.out.println("EntryPoint.producer.index");
+        String result = "";
+        String messageId = "";
+        String res = null ;
+
+        Map<String, String> map = new HashMap<>();
+        map.put("message", "my new message post ");
+        map.put("User-Agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X)");
+        try {
+            res = kafkaPublisher.produce(map);
+        } catch (Exception e){
```

Comment:
Don't catch exceptions if you don't have to.
It is messing the code and makes it hard to read